extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -148,7 +148,7 @@ export abstract class DacFxConfigPage extends BasePage {
 	}
 
 	protected generateFilePathFromDatabaseAndTimestamp(): string {
-		return path.join(this.getRootPath(), sanitizeStringForFilename(this.model.database) + '-' + this.getDateTime() + this.fileExtension);
+		return this.model.database ? path.join(this.getRootPath(), sanitizeStringForFilename(this.model.database) + '-' + this.getDateTime() + this.fileExtension) : '';
```

Comment:
this is assuming this.model is not null - does the assumption stand always?

Reply:
yes it gets initialized when the wizard is started. There isn't anything that sets the model to null

Reply:
The constructor checks for null? or its never created with null?